docs/dump_javadump.md

```diff
@@ -436,68 +436,73 @@ NULL           -----------------------------------------------------------------
 NULL           =================================
 NULL
 1XMPOOLINFO    JVM Thread pool info:
-2XMPOOLTOTAL       Current total number of pooled threads: 18
-2XMPOOLLIVE        Current total number of live threads: 16
+2XMPOOLTOTAL       Current total number of pooled threads: 19
+2XMPOOLLIVE        Current total number of live threads: 18
 2XMPOOLDAEMON      Current total number of live daemon threads: 15
 NULL
 1XMTHDINFO     Thread Details
 NULL
-3XMTHREADINFO      "JIT Diagnostic Compilation Thread-7 Suspended" J9VMThread:0x0000000000EFC500, omrthread_t:0x00007FF4F00A77E8, java/lang/Thread:0x00000000FFE97480, state:R, prio=10
-3XMJAVALTHREAD            (java/lang/Thread getId:0xA, isDaemon:true)
-3XMTHREADINFO1            (native thread ID:0x7657, native priority:0xB, native policy:UNKNOWN, vmstate:CW, vm thread flags:0x00000081)
-3XMTHREADINFO2            (native stack address range from:0x00007FF4CCC36000, to:0x00007FF4CCD36000, size:0x100000)
-3XMCPUTIME               CPU usage total: 0.000037663 secs, current category="JIT"
+...
+3XMTHREADINFO      "JIT Diagnostic Compilation Thread-007 Suspended" J9VMThread:0x0000000000035200, omrthread_t:0x00007F3F8C0D02C8, java/lang/Thread:0x00000000FFF42120, state:R, prio=10
+3XMJAVALTHREAD            (java/lang/Thread getId:0x9, isDaemon:true)
+3XMJAVALTHRCCL            sun/misc/Launcher$AppClassLoader(0x00000000FFF3BF98)
+3XMTHREADINFO1            (native thread ID:0x618F, native priority:0xB, native policy:UNKNOWN, vmstate:CW, vm thread flags:0x00000081)
+3XMTHREADINFO2            (native stack address range from:0x00007F3F879C5000, to:0x00007F3F87AC5000, size:0x100000)
+3XMCPUTIME               CPU usage total: 0.052410771 secs, current category="JIT"
 3XMHEAPALLOC             Heap bytes allocated since last GC cycle=0 (0x0)
 3XMTHREADINFO3           No Java callstack associated with this thread
-3XMTHREADINFO3           No native callstack available for this thread
+...
```

Comment:
I think it would be sufficient to add sample `3XMJAVALTHRCCL` lines; the other changes are not necessary.

Reply:
Since this is a sample and is not expected to be reused, focus was on the change in the 3XMTHREADBLOCK line and the existing information mentioned in the section. Also, sample was updated with the relevant changes as seen in the latest java dump while keeping it as close to the earlier sample as possible. Therefore, we can leave this change as-is unless you foresee any problems.